docs/source/bounded_variables.rst

```diff
@@ -0,0 +1,54 @@
+=================
+Bounded Variables
+=================
+
+Some variables have constraints.  For instance, it makes no sense for a
+standard deviation to have a negative value, so something like a Normal prior
+on a parameter that represents a standard deviation would be inappropriate.
+PyMC3 includes distributions that have positive valued support, such as
+``Gamma`` or ``Exponential``.  PyMC3 also includes several bounded
+distributions, such as ``Uniform``, ``HalfNormal``, and ``HalfCauchy``, that
+are restricted to a specific domain.  Variables that have been assigned these
+distributions are automatically transformed to remove the discontinuity of the
+probability distribution introduced at zero. 
+
+PyMC3 includes the construct ``Bound`` for placing constraints on existing
+distributions.  For instance, one may have prior information that suggests that
+parameter representing a standard deviation is near one.  One could use a
+Normal distribution while constraining the support to positive values.  A
+``BoundedNormal`` distribution can be defined before or within the model block.
```

Comment:
I think it might be safer to always but the definition in the model context. We can run into trouble with the test values otherwise (I really hope we can get rid of those at some point...). It should still be on a separate line though.